### ISSUE TITLE
Canceled repeating events still show up in the dashboard widget as happening

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/calendar#2634]
### Dashboard
![Bildschirmfoto von 2020-10-08 10-47-21](https://user-images.githubusercontent.com/213943/95436228-e55dd700-0953-11eb-9e73-f65d6175516b.png)

### Calendar view

![Bildschirmfoto von 2020-10-08 10-47-32](https://user-images.githubusercontent.com/213943/95436277-f1e22f80-0953-11eb-8755-8a3960913ad7.png)

### Steps to reproduce
1. Create a repeating event
2. Cancel the one occurance of tomorrow
3. See Item still being "active" in the dashboard

### Calendar app
**Calendar app version:** 2.1.2

### Server configuration

**Nextcloud Version:** 20.0.0

**List of activated apps:**

```
  - accessibility: 1.6.0
  - activity: 2.13.1
  - analytics: 3.0.0
  - calendar: 2.1.2
  - cloud_federation_api: 1.3.0
  - comments: 1.10.0
  - contacts: 3.4.0
  - contactsinteraction: 1.1.0
  - dashboard: 7.0.0
  - dav: 1.16.0
  - deck: 1.1.0
  - dropit: 0.4.0
  - event_update_notification: 1.1.0
  - federatedfilesharing: 1.10.1
  - federation: 1.10.1
  - files: 1.15.0
  - files_accesscontrol: 1.10.0
  - files_automatedtagging: 1.10.0
  - files_markdown: 2.3.1
  - files_pdfviewer: 2.0.1
  - files_rightclick: 0.17.0
  - files_sharing: 1.12.0
  - files_trashbin: 1.10.1
  - files_versions: 1.13.0
  - files_videoplayer: 1.9.0
  - firstrunwizard: 2.9.0
  - logreader: 2.5.0
  - lookup_server_connector: 1.8.0
  - mail: 1.5.0
  - nextcloud_announcements: 1.9.0
  - notifications: 2.8.0
  - oauth2: 1.8.0
  - password_policy: 1.10.1
  - photos: 1.2.0
  - privacy: 1.4.0
  - provisioning_api: 1.10.0
  - richdocuments: 3.7.4
  - richdocumentscode: 4.2.800
  - settings: 1.2.0
  - sharebymail: 1.10.0
  - spreed: 10.0.0
  - support: 1.3.0
  - survey_client: 1.8.0
  - systemtags: 1.10.0
  - tasks: 0.13.3
  - text: 3.1.0
  - theming: 1.11.0
  - twofactor_backupcodes: 1.9.0
  - updatenotification: 1.10.0
  - user_status: 1.0.0
  - viewer: 1.4.0
  - weather_status: 1.0.0
  - workflowengine: 2.2.0
```

**Comment by graue70** [comment]
This is independent of repeating events, but is instead true for any canceled event. The title and steps to reproduce could therefore be simplified.

The widget also doesn't differentiate between 'tentative' and 'confirmed' events (the other two categories besides 'canceled').

Also, for somebody looking to fix this. This might be a good place to start: https://github.com/nextcloud/calendar/blob/master/src/views/Dashboard.vue#L226

**Comment by tcitworld** [comment]
I think we can agree on hiding cancelled events, but what about tentative ones? There's not much space and not many possibilities for UI indications that the event is tentative. Should we just hide them as well?

**Comment by graue70** [comment]
I thought the solution was not to hide canceled events but instead show events like they are shown in the calendar app itself:
![grafik](https://user-images.githubusercontent.com/23035329/104498733-63c5b700-55dc-11eb-8621-ddd986df1806.png)

I guess I would do that for tentative events no matter whether canceled events are hidden or not. I would definitely NOT hide tentative events.

**Comment by nickvergessen** [comment]
The main issue is that the `DashboardWidgetItem` component from the `DashboardWidget` library does not allow further text styling atm.

**Comment by graue70** [comment]
> The main issue is that the `DashboardWidgetItem` component from the `DashboardWidget` library does not allow further text styling atm.

In that case I agree that the quickfix is to hide canceled events altogether and to ignore the differentiation between tentative and confirmed events for now.

**Comment by elhananjair** [comment]
@ChristophWurst this issue is coming back, I am trying with Calendar Appointment. 
I set up an appointment and I confirmed it in my email. Then the organizer declined the event from the calendar, but the event is still coming up on the dashboard upcoming event and is also shown on the Calendar as an Active event. I am on Nextcloud 24.0.6

![Screenshot from 2022-10-24 08-27-58](https://user-images.githubusercontent.com/69236638/197534516-b902c833-8e50-42c1-9362-b10a9e3748cf.png)
---
![Screenshot from 2022-10-24 08-27-17](https://user-images.githubusercontent.com/69236638/197534576-2396e6ef-45a3-4560-93e3-1f8dc1eed6b4.png)
---
![Screenshot from 2022-10-24 08-26-42](https://user-images.githubusercontent.com/69236638/197534634-77d59442-8862-4f6d-8bd3-8d298bc01d1f.png)
